src/xmllang/xml_parser.h

```diff
@@ -28,29 +42,27 @@ class xml_parsert:public parsert
     return *stack.back();
   }
 
-  virtual bool parse()
-  {
-    return yyxmlparse()!=0;
-  }
+  bool parse() override;
 
   void new_level()
   {
     current().elements.push_back(xmlt());
     stack.push_back(&current().elements.back());
   }
 
-  virtual void clear()
+  void clear() override
```

Comment:
❓ Same question as on #8141: Is this still needed? Should it be renamed to init and made private? (Maybe for a future PR)

Reply:
See https://github.com/diffblue/cbmc/pull/8141#discussion_r1526102487.